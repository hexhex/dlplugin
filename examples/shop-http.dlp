% -*- prolog -*-

plusProvides("provides","foo","moo").
plusPart("Part","cpu1").
minusPart("Part","cpu2").
minusShop("Shop","not_a_shop").

s(X) :- &dlC["http://www.kr.tuwien.ac.at/staff/roman/swlp/examples/shop.owl",plusPart,minusPart,plusProvides,foo,"Shop"](X).
p(X,Y) :- &dlR["http://www.kr.tuwien.ac.at/staff/roman/swlp/examples/shop.owl",plusPart,minusPart,plusProvides,foo,provides](X,Y).
part(X) :- &dlC["http://www.kr.tuwien.ac.at/staff/roman/swlp/examples/shop.owl",plusPart,minusPart,plusProvides,foo,"Part"](X).

nshop(X) :- &dlC["http://www.kr.tuwien.ac.at/staff/roman/swlp/examples/shop.owl",plusPart,minusShop,plusProvides,foo,"-Shop"](X).
