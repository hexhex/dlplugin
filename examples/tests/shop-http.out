{nshop("<http://www.kr.tuwien.ac.at/staff/roman/shop#not_a_shop>"), part("<http://www.kr.tuwien.ac.at/staff/roman/shop#cpu1>"), part("<http://www.kr.tuwien.ac.at/staff/roman/shop#moo>"), part("<http://www.kr.tuwien.ac.at/staff/roman/shop#fan>"), part("<http://www.kr.tuwien.ac.at/staff/roman/shop#wlan>"), part("<http://www.kr.tuwien.ac.at/staff/roman/shop#graphiccard>"), part("<http://www.kr.tuwien.ac.at/staff/roman/shop#cdrom>"), part("<http://www.kr.tuwien.ac.at/staff/roman/shop#harddisk>"), part("<http://www.kr.tuwien.ac.at/staff/roman/shop#dvdrom>"), part("<http://www.kr.tuwien.ac.at/staff/roman/shop#memory>"), part("<http://www.kr.tuwien.ac.at/staff/roman/shop#soundcard>"), part("<http://www.kr.tuwien.ac.at/staff/roman/shop#cpu>"), part("<http://www.kr.tuwien.ac.at/staff/roman/shop#case>"), p("<http://www.kr.tuwien.ac.at/staff/roman/shop#s1>","<http://www.kr.tuwien.ac.at/staff/roman/shop#cpu>"), p("<http://www.kr.tuwien.ac.at/staff/roman/shop#s1>","<http://www.kr.tuwien.ac.at/staff/roman/shop#case>"), p("<http://www.kr.tuwien.ac.at/staff/roman/shop#s2>","<http://www.kr.tuwien.ac.at/staff/roman/shop#soundcard>"), p("<http://www.kr.tuwien.ac.at/staff/roman/shop#s3>","<http://www.kr.tuwien.ac.at/staff/roman/shop#dvdrom>"), p("<http://www.kr.tuwien.ac.at/staff/roman/shop#s3>","<http://www.kr.tuwien.ac.at/staff/roman/shop#memory>"), p("<http://www.kr.tuwien.ac.at/staff/roman/shop#s3>","<http://www.kr.tuwien.ac.at/staff/roman/shop#cpu>"), p("<http://www.kr.tuwien.ac.at/staff/roman/shop#s4>","<http://www.kr.tuwien.ac.at/staff/roman/shop#cdrom>"), p("<http://www.kr.tuwien.ac.at/staff/roman/shop#s4>","<http://www.kr.tuwien.ac.at/staff/roman/shop#harddisk>"), p("<http://www.kr.tuwien.ac.at/staff/roman/shop#s8>","<http://www.kr.tuwien.ac.at/staff/roman/shop#cdrom>"), p("<http://www.kr.tuwien.ac.at/staff/roman/shop#s8>","<http://www.kr.tuwien.ac.at/staff/roman/shop#dvdrom>"), p("<http://www.kr.tuwien.ac.at/staff/roman/shop#s8>","<http://www.kr.tuwien.ac.at/staff/roman/shop#memory>"), p("<http://www.kr.tuwien.ac.at/staff/roman/shop#s8>","<http://www.kr.tuwien.ac.at/staff/roman/shop#soundcard>"), p("<http://www.kr.tuwien.ac.at/staff/roman/shop#s5>","<http://www.kr.tuwien.ac.at/staff/roman/shop#harddisk>"), p("<http://www.kr.tuwien.ac.at/staff/roman/shop#s5>","<http://www.kr.tuwien.ac.at/staff/roman/shop#cpu>"), p("<http://www.kr.tuwien.ac.at/staff/roman/shop#s6>","<http://www.kr.tuwien.ac.at/staff/roman/shop#graphiccard>"), p("<http://www.kr.tuwien.ac.at/staff/roman/shop#s6>","<http://www.kr.tuwien.ac.at/staff/roman/shop#case>"), p("<http://www.kr.tuwien.ac.at/staff/roman/shop#s7>","<http://www.kr.tuwien.ac.at/staff/roman/shop#wlan>"), p("<http://www.kr.tuwien.ac.at/staff/roman/shop#s7>","<http://www.kr.tuwien.ac.at/staff/roman/shop#harddisk>"), p("<http://www.kr.tuwien.ac.at/staff/roman/shop#s9>","<http://www.kr.tuwien.ac.at/staff/roman/shop#harddisk>"), p("<http://www.kr.tuwien.ac.at/staff/roman/shop#s9>","<http://www.kr.tuwien.ac.at/staff/roman/shop#case>"), p("<http://www.kr.tuwien.ac.at/staff/roman/shop#s10>","<http://www.kr.tuwien.ac.at/staff/roman/shop#fan>"), p("<http://www.kr.tuwien.ac.at/staff/roman/shop#foo>","<http://www.kr.tuwien.ac.at/staff/roman/shop#moo>"), s("<http://www.kr.tuwien.ac.at/staff/roman/shop#s1>"), s("<http://www.kr.tuwien.ac.at/staff/roman/shop#s2>"), s("<http://www.kr.tuwien.ac.at/staff/roman/shop#s3>"), s("<http://www.kr.tuwien.ac.at/staff/roman/shop#s4>"), s("<http://www.kr.tuwien.ac.at/staff/roman/shop#s8>"), s("<http://www.kr.tuwien.ac.at/staff/roman/shop#s5>"), s("<http://www.kr.tuwien.ac.at/staff/roman/shop#s6>"), s("<http://www.kr.tuwien.ac.at/staff/roman/shop#s7>"), s("<http://www.kr.tuwien.ac.at/staff/roman/shop#s9>"), s("<http://www.kr.tuwien.ac.at/staff/roman/shop#s10>"), s("<http://www.kr.tuwien.ac.at/staff/roman/shop#foo>"), minusShop("Shop","not_a_shop"), plusProvides("provides","foo","moo"), minusPart("Part","cpu2"), plusPart("Part","cpu1")}
